tests/conftest.py

```diff
@@ -258,6 +261,26 @@ def ptfhost(ansible_adhoc, tbinfo, duthost):
         ptf_host = duthost.host.options["inventory_manager"].get_host(duthost.hostname).get_vars()["ptf_host"]
         return PTFHost(ansible_adhoc, ptf_host)
 
+@pytest.fixture(scope="module")
+def k8shosts(ansible_adhoc, request, creds):
+    """
+    Shortcut fixture for getting Kubernetes hosts
+    """
+    master_id = request.config.getoption("--kube_master") 
+    # master_id is required to be in the format {servernumber}_{mastersetid} where 2-digit servernumber is enforced with leading 0 when necessary
+    master_server_id = int(master_id[:2])
+    master_set_id = int(master_id[3:]) 
+    k8s_ansible_group = "k8s_vms{}_{}".format(master_set_id, master_server_id)
+    master_vms = {}
+    with open('../ansible/k8s-ubuntu', 'r') as kinv:
```

Comment:
>.. [](start = 15, length = 2)

The path is relative to the current working directory, not the script's directory.

Reply:
This is similar to the way a specific file is opened in other parts of the `tests/conftest.py` file as well
eg `with open("../ansible/vars/topo_{}.yml".format(topo), 'r') as fh:`